src/Display.js

```diff
@@ -5,12 +5,12 @@ import ListOfWinners from "./components/ListOfWinners";
 import {QuizContainer} from "./QuizContainer";
 
 function Display() {
-    const {quizStatus, registeredUser} = useContext(QuizContainer.QuizContext);
+    const {quizStatus, winnersList} = useContext(QuizContainer.QuizContext);
     const quizScheduled = quizStatus === "scheduled";
     const gameInProgress = quizStatus === "started";
-    const showListOfWinners = quizStatus === "finished";
+    const showListOfWinners = winnersList.length > 0;
```

Comment:
zastanawiam się nad tym warunkiem, bo jak "gra" tylko spectator to nigdy quiz się nie zakończy 🤔 może w wypadku pustej tabelki wyświetlić "listę wyników" z napisem że nikt nie wygrał czy coś? a wtedy by trzeba było zmienić ten warunek, żeby nie polegać na winnersList.length

Reply:
można np pobierać boolean `showListOfWinners` bezpośrednio z QuizContextu, a będzie on ustawiany na true przez `socket.on('winners_list',...)`